src/pages/Home/components/CardsMenu/index.tsx

```diff
@@ -99,18 +146,41 @@ function CardsMenu() {
                 />
               </picture>
             </StyledCardMedia>
-
-            <StyledCardContent>
-              <StyledHeading as="h4">
-                {t(card.title)}
-                {card.target === '_blank' ? (
-                  <Icon>
-                    <ExternalLink />
-                  </Icon>
-                ) : (
-                  <></>
+            <StyledCardContent
+              onClick={() => setShowTooltip(showTooltip ? '' : card.path)}
+              onMouseOver={() => setShowTooltip(card.path)}
```

Comment:
Bepaalde gegevens alleen tonen bij een mouse-over is niet heel toegankelijk - dit heb je bijvoorbeeld niet op touch-screens.

Reply:
See my earlier comment; we don't expect users on mobile to see these descriptions. The descriptions are available in `title` attributes in the header and as `alt` text in the tile's images.